src/ansys/fluent/core/post_objects/post_objects_container.py

```diff
@@ -0,0 +1,126 @@
+"""Module providing visualization objects for Matplotlib."""
+import inspect
+
+from ansys.fluent.core.meta import PyLocalContainer
+
+
+class Container:
+    def __init__(self, session, child, module, local_surfaces_provider=None):
+        """Instantiate Plots, container of plot objects.
+
+        Parameters
+        ----------
+        session :
+            Session object.
+        local_surfaces_provider : object, optional
+            Object providing local surfaces.
+        """
+        session_state = child._sessions_state.get(session.id if session else 1)
+        if not session_state:
+            session_state = self.__dict__
+            child._sessions_state[session.id if session else 1] = session_state
+            self.session = session
+            self._init_module(self, module)
+        else:
+            self.__dict__ = session_state
+        self._local_surfaces_provider = lambda: local_surfaces_provider or getattr(
+            self, "Surfaces", []
+        )
+
+    def _init_module(self, obj, mod):
+        from ansys.fluent.core.post_objects.post_helper import PostAPIHelper
+
+        for name, cls in mod.__dict__.items():
+
+            if cls.__class__.__name__ in (
+                "PyLocalNamedObjectMetaAbstract",
+            ) and not inspect.isabstract(cls):
+                setattr(
+                    obj,
+                    cls.PLURAL,
+                    PyLocalContainer(self, cls, PostAPIHelper),
+                )
+
+
+class Plots(Container):
```

Comment:
@prmukherj  Should we rename to Plots and Graphics to PlotsContainer and GraphicsContainer.
Instaed of calling PlotsBase and GraphicsBase,